.github/workflows/pathogen-repo-ci.yaml.in

```diff
@@ -89,6 +89,16 @@ on:
         default: false
         required: false
 
+  workflow_dispatch:
+    inputs:
+      <<: *inputs
+      repo:
+        description: >-
+          Repository name with owner (e.g. nextstrain/zika).
+        type: string
+        default: ""
+        required: true
+
```

Comment:
> I don't understand why it needs to be generated from a template? What's going on with that?

@genehack The use of the template is most visible in this change. Instead of having to redefine the same inputs again for `workflow_dispatch` to match `workflow_call`, just point to the YAML anchor.